### DISCUSSION TITLE
How can we use Cucumber tags

### DISCUSSION THREAD

**Discussion by DevPerers** [badeball/cypress-cucumber-preprocessor#986] [General]
How can we filter which test case to be executed by Cucumber tag

**Comment by badeball** [comment]
https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/tags.md

**Comment by DevPerers** [comment]
I tried but not works.
I created a Cypress test with Cucumber, written in Java Scripts.
I want to use Cucumber tags, But when I try to run by following command its open difference scenarios, its seems like its start to execute all the scenarios.
npx cypress run --env tags=@smoke --browser chrome --headless
I place @smoke tag above one scenario in a feature file

**Reply by badeball** [reply]
This isn’t a support forum, as stated in the contribution guidelines. That doesn’t necessarily mean I can’t help you though. However, you’re making me chase down the relevant information, making this more difficult than it has to be. You need to read the section about bug reports and try again: https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/CONTRIBUTING.md#bug-reports

**Comment by DevPerers** [comment]
Its working fine
